src/agent/model_util.h

```diff
@@ -670,6 +675,12 @@ inline bool operator!= (const TimestampModel& t1, const TimestampModel& t2) {
   return !(t1 == t2);
 }
 
+inline TimestampModel GetCreateTimestamp(const BreakpointModel& model) {
+  return (model.create_time != kUnspecifiedTimestamp)
+             ? model.create_time
+             : model.create_time_unix_msec;
+}
+
```

Comment:
Nit: I always find helper methods like this to be a bit frustrating; it's non-obvious as to where the implementation is when viewing it from where it is used (ie. jvm_breakpoint.cc).  I'd prefer that this either be part of the BreakpointModel or in the jvm_breakpoint file, but accept that this is just personal preference.

Reply:
I initially wanted to put it in the BreakpointModel, the issue with that was kUnspecifiedTimestamp and the TimestampModel comparison operators are defined here in model_util. To help, I've introduced a model_util namespace to wrap this helper method.